app/src/main/java/com/cornellappdev/uplift/MainActivity.kt

```diff
@@ -3,14 +3,21 @@ package com.cornellappdev.uplift
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
+import com.cornellappdev.uplift.ui.MainNavigationWrapper
 import com.cornellappdev.uplift.ui.theme.UpliftTheme
+import com.cornellappdev.uplift.ui.viewmodels.GymDetailViewModel
+import com.cornellappdev.uplift.util.testMorrison
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        val gymDetailViewModel = GymDetailViewModel()
+
+        gymDetailViewModel.selectGym(testMorrison)
+
```

Comment:
Sorry small comment, viewmodels should not be created like this but should be referenced using viewModel(), see[ this link](https://developer.android.com/jetpack/compose/libraries#viewmodel) for an example.

viewModel() returns an existing ViewModel or creates a new one in the given scope. The ViewModel is retained as long as the scope is alive. For example, if the composable is used in an activity, viewModel() returns the same instance until the activity is finished or the process is killed.